reverse_engineering/helpers/getSchemaSequences.js

```diff
@@ -115,14 +122,22 @@ const getSchemaSequenceDtos = async ({ schema, execute, logger }) => {
 			return [];
 		}
 
+		const usedSequences = filterUsedSequences({ sequenceDtos: rawSequenceDtos, allDDLs });
+
+		logger.log(
+			'info',
+			{ message: 'Filter sequences finished', usedSequencesCount: usedSequences?.length || 0 },
```

Comment:
I guess this means 'Filtering of sequences used in tables and views finished', at first I saw Getting sequences (which is the title i guess) and it was a bit misleading.